src/include/block_chunkcache.h

```diff
@@ -40,6 +40,11 @@ struct __wt_chunkcache_chunk {
     wt_off_t chunk_offset;
     size_t chunk_size;
     volatile uint32_t valid;
+
+/* AUTOMATIC FLAG VALUE GENERATION START 0 */
+#define WT_CHUNK_PINNED 0x1u
+    /* AUTOMATIC FLAG VALUE GENERATION STOP 8 */
```

Comment:
Nitpick, indentation seems messed up here

Reply:
The formatting is usually handled by `s_all`. After I fix it, `s_all` chooses to indent it. I have seen it in different places, for example, [here](https://github.com/wiredtiger/wiredtiger/blob/develop/src/include/btmem.h#L169).

Reply:
Fair enough - I'll resolve this.